[docs][charts] Similar introduction on most charts pages (#17374)
mui/mui-x: docs/data/charts/heatmap/heatmap.md

## Basics

> Heatmap charts series should contain a `data` property containing an array of 3-tuples.

Reviewer: ```suggestion
Heatmap charts series must contain a `data` attribute containing an array of three tuples.
```

> The 2 first numbers are respectively the x and y indexes of the cell.

Reviewer: ```suggestion
The first two numbers in each tuple correspond to the x and y indexes of the cell, respectively.
```

> And the third is its value.

Reviewer: ```suggestion
The third number is the value for the given cell.
```

```jsx
series={[{
    data: [

>         [0, 2, 2.7], // Cell (0, 2) get the value 2.7

Reviewer: ```suggestion
        [0, 2, 2.7], // Cell (0, 2) receives the value 2.7
```

>         [1, 2, 4.5], // Cell (1, 2) get the value 4.5

Reviewer: ```suggestion
        [1, 2, 4.5], // Cell (1, 2) receives the value 4.5
```
